Source/Core/UICommon/DiscordPresence.cpp

```diff
@@ -15,20 +19,69 @@
 
 namespace Discord
 {
+#ifdef USE_DISCORD_PRESENCE
+static JoinFunction join_function = nullptr;
```

Comment:
This should likely be structured in such a way that file-static state isn't necessary